src/test/kotlin/org/rust/ide/annotator/RsEdition2018KeywordsAnnotatorTest.kt

```diff
@@ -94,7 +94,9 @@ class RsEdition2018KeywordsAnnotatorTest : RsAnnotatorTestBase(RsEdition2018Keyw
             () => { async };
         }
 
+        #[async, async(async, async)]
         fn main() {
+            #![async, async(async, async)]
```

Comment:
Probably, it's worth check `foo::async` paths in attributes as well

Reply:
I have fixed the previous solution because `async` cannot be used as an attbitute name:

<img width="656" alt="Screenshot 2021-05-26 at 17 57 21" src="https://user-images.githubusercontent.com/6079006/119683155-1986f900-be4c-11eb-988b-d6eb955fd077.png">

Reply:
@mchernyavsky let's add the corresponding test